doc/classes/Curve2D.xml

```diff
@@ -107,16 +107,14 @@
 			<param index="0" name="offset" type="float" default="0.0" />
 			<param index="1" name="cubic" type="bool" default="false" />
 			<description>
-				Similar to [method sample_baked], but returns [Transform2D] that includes a rotation along the curve, with [member Transform2D.origin] as the point position, [member Transform2D.x] as the sideways vector, and [member Transform2D.y] as the forward vector. Returns an empty transform if the length of the curve is [code]0[/code].
+				Similar to [method sample_baked], but returns [Transform2D] that includes a rotation along the curve, with [member Transform2D.origin] as the point position and the [member Transform2D.x] vector pointing in the direction of the path at that point. Returns an empty transform if the length of the curve is [code]0[/code].
 				[codeblock]
-				var baked = curve.sample_baked_with_rotation(offset)
-				# This will rotate and position the node with the up direction pointing along the curve.
+				var baked := curve.sample_baked_with_rotation(offset)
```

Comment:
We don't typically use type hints in the documentation examples, to keep them simple.